packages/btp-utils/src/app-studio.ts

```diff
@@ -84,3 +84,14 @@ export async function listDestinations(): Promise<Destinations> {
     });
     return destinations;
 }
+
+/**
+ * Exposes port in SAP Business Application Studio.
+ *
+ * @param port Port that needs to be exposed
+ * @returns url on which the port is exposed
+ */
+export async function exposePort(port: number): Promise<string> {
+    const response = await axios.get(`http://localhost:3001/AppStudio/api/getHostByPort?port=${port}`);
+    return `${response.data.result}`;
```

Comment:
Please add `try/catch` for the cases when the API is not available. The exception should not block the process of starting the local server.